README.md

```diff
@@ -105,6 +105,10 @@ Software tools or libraries, sorted by programming language.
 
 Data viewers/Editors based on shapes.
 
+- [form-generator](https://git.rwth-aachen.de/coscine/frontend/libraries/form-generator) <img alt="NPM Version" src="https://img.shields.io/npm/v/@coscine/form-generator" align="top"> - A Vue SHACL form generator library; `MIT` license; `Typescript`.
+- [shacl-form](https://github.com/danielbeeke/shacl-form) <img alt="GitHub Release Date" src="https://img.shields.io/github/release-date/danielbeeke/shacl-form" align="top"> - A custom element SHACL form generator library; [docs](https://shacl-form.mediaworks.global/); `MIT` license; `Typescript` ☠️.
```

Comment:
Since the 3 projects sound so similar, please add a qualifier: 
- shacl-form (DanielBeeke)
- shacl-form (ULB-Darmstadt)
- shacl-forms (Zazuko)

Reply:
@bergos please don't give up in frustration!
I think you'll agree with my comment here: we need to distinguish the 2 projects , by using a qualifier